docs/src/components/ComponentDoc/ComponentControls/ComponentButton.tsx

```diff
@@ -0,0 +1,31 @@
+import { createComponent, Icon, Text, ICSSInJSStyle } from '@stardust-ui/react'
+import * as React from 'react'
+
+interface LabelledButtonProps {
+  iconName: string
+  label: string
+  active: boolean
+  onClick?: (event: React.SyntheticEvent) => void
+}
+
+const centered: ICSSInJSStyle = {
+  textAlign: 'center',
+}
+
+const LabelledButton: React.SFC<LabelledButtonProps> = createComponent<LabelledButtonProps>({
+  displayName: 'LabelledButton',
```

Comment:
For tracking:

We can introduce variables for having different Button variants in the docs theme. I am pretty confident that the API of the Button at least for now will be enough for achieving similar look.